Tools/ssl/multissltests.py

```diff
@@ -453,6 +465,34 @@ class BuildLibreSSL(AbstractBuilder):
     build_template = "libressl-{}"
 
 
+class BuildAWSLC(AbstractBuilder):
+    library = "AWS-LC"
+    url_templates = (
+        "https://github.com/aws/aws-lc/archive/refs/tags/v{v}.tar.gz",
+    )
+    src_template = "aws-lc-{}.tar.gz"
+    build_template = "aws-lc-{}"
+
+    def _build_src(self, config_args=()):
+        cwd = self.build_dir
+        log.info("Running build in {}".format(cwd))
+        env = os.environ.copy()
+        env["LD_RUN_PATH"] = self.lib_dir # set rpath
+        if self.system:
+            env['SYSTEM'] = self.system
+        cmd = [
+            "cmake",
+            "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+            "-DCMAKE_PREFIX_PATH={}".format(self.install_dir),
+            "-DCMAKE_INSTALL_PREFIX={}".format(self.install_dir),
+            "-DBUILD_SHARED_LIBS=ON",
+            "-DBUILD_TESTING=OFF",
+            "-DFIPS=OFF",
+        ]
+        self._subprocess_call(cmd, cwd=cwd, env=env)
+        self._subprocess_call(["make", f"-j{self.jobs}"], cwd=cwd, env=env)
```

Comment:
```suggestion
        self._subprocess_call(["make", "-j{}".format(self.jobs)], cwd=cwd, env=env)
```

Alternatively, it's probably time that we could remove 2.7 from the list of versions we should stay compatible with.

Reply:
> Alternatively, it's probably time that we could remove 2.7 from the list of versions we should stay compatible with.

That's certainly a call for the maintainers. No concerns from AWS-LC's perspective as the earliest version we [test against](https://github.com/aws/aws-lc/blob/main/tests/ci/integration/python_patch/3.9/aws-lc-cpython.patch) is 3.9.

Reply:
@zware -- the 2.7 compat issues didn't show up in this PR's CI. is there some other slack/zulip/etc. channel where i can monitor secondary CI builds to get ahead of these issues during review?

Reply:
I personally have been ready to drop 2.7 compatibility everywhere for over a decade now :).  I have no evidence that anyone is still actually using 2.7 with this script; that note is [8 years old](https://github.com/python/cpython/blame/69f69e6b68b945bde7d6c4dbe3efe344e8d1ac22/Tools/ssl/multissltests.py#L21), from back when we were still maintaining each of the mentioned versions.